packages/mendel-outlet-browser-pack/src/index.js

```diff
@@ -65,10 +72,12 @@ module.exports = class BrowserPackOutlet {
             let prelude = '';
             let appendix = '';
 
-            if (hasProcess) {
-                prelude = '(function(){var process={env: {}};';
+            if (hasProcess || hasGlobal) {
+                prelude = '(function(){';
                 appendix = '})();';
             }
+            if (hasProcess) prelude += 'var process={env: {}};';
+            if (hasGlobal) prelude += 'var global=window;';
```

Comment:
I can merge first and change later if you are time constrained today but you agree it is a good idea.